### PR TITLE
 Ir/ednx/ha 2

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

File: lms/envs/ednx_prod.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# We intentionally define lots of variables that aren't used, and
+# want to import all variables from base settings files
+# pylint: disable=wildcard-import, unused-wildcard-import
+
+from .production import *
+
+
+# Reading setting to skip or not the enrollment start date filtering for edx-search
+SEARCH_SKIP_ENROLLMENT_START_DATE_FILTERING = ENV_TOKENS.get(
```
Comment (morenol) [inline lms/envs/ednx_prod.py]: I think we should add this to upstream and remove from this commit @jfavellar90

Comment (felipemontoya) [reply lms/envs/ednx_prod.py]: This is actually not very useful now, since we can alter this settings during runtime using eox-tenant

**Comment by morenol** [comment]
Since we are using now the plugins to override the settings as we want, does it make sense to have the ednx_prod.py files? @felipemontoya @jfavellar90

File: cms/envs/ednx_devstack.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+""" Overrides for Docker-based eduNEXT devstack. """
+
+# We intentionally define lots of variables that aren't used, and
+# want to import all variables from base settings files
+# pylint: disable=wildcard-import, unused-wildcard-import
+
+from .devstack_docker import *
```
Comment (jfavellar90) [inline cms/envs/ednx_devstack.py]: Do we run our devstack using these settings??

File: lms/envs/ednx_devstack.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+""" Overrides for Docker-based eduNEXT devstack. """
```
Comment (jfavellar90) [inline lms/envs/ednx_devstack.py]: Do we run our devstack using these settings??

**Comment by jfavellar90** [comment]
@morenol in fact I don't know why we have those ednx_prod.py files. Are we running from these settings in production? and yeah, with plugin we can override what we want. cc @felipemontoya

File: cms/envs/ednx_devstack.py (hunk 2 above)
Comment (morenol) [reply cms/envs/ednx_devstack.py]: Yes, but in the case of CMS it is not strictly necessary

File: lms/envs/ednx_devstack.py (hunk 3 above)
Comment (morenol) [reply lms/envs/ednx_devstack.py]: Yes, we do. We need this for the integration with the other edunext services

**Comment by jfavellar90** [comment]
@morenol it looks good to me

**Comment by morenol** [comment]
@jfavellar90 I am going to squash and wait for the tests to merge this.

**Pull request merged**